app/services/create_user_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateUserService
+  def initialize(params)
+    @params = params.dup
+  end
+
+  def run!
+    user_type = case @params[:userType]
+                when 'driver'
+                  User::Driver
+                when 'passenger'
+                  User::Passenger
+                else
+                  return
+                end
```

Comment:
case문을 사용하지 않아도 되는 아래와 같은 패턴도 있습니다.

```suggestion
    USER_TYPES_CLASS = {
      'driver' => User::Driver,
      'passenger' => User::Passenger
    }
    ...
    user_type = USER_TYPES_CLASS[@params[:userType]]
```

Reply:
요거 재사용 가능성이 있다고 생각들어서 User 모델에 구현하는 방법으로 [9f9ac24](https://github.com/travelingaries/taxi-dispatch-backend-api/pull/1/commits/9f9ac24102f3e27ec4faf23c52b186901261ad9a)에서 수정했어요~